package.json

```diff
@@ -23,15 +23,17 @@
     "postshrinkwrap": "test -z $CI && ./scripts/packagelockHttps.sh; git update-index --assume-unchanged .env",
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -s .storybook/static -c .storybook",
-    "test": "npm run test:lint && npm run test:dependencies && npm run test:unit && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
-    "test:ci": "npm run start & npm install --no-save cypress@3.4.1 bbc-a11y@latest npm-run-all@latest puppeteer@latest audit-ci@latest && run-p cypress:ci test:puppeteer && run-p bbcA11y:ci lighthouse && audit-ci --low --whitelist acorn minimist",
+    "test": "npm run test:lint && npm run test:dependencies && npm run test:unit && npm run build -- --hide-modules --colors && run-p --race start amp:validate && run-p --race start test:integration",
+    "test:ci": "npm run start & npm install --no-save cypress@3.4.1 bbc-a11y@latest npm-run-all@latest puppeteer@latest audit-ci@latest && npm run test:integration && run-p cypress:ci test:puppeteer && run-p bbcA11y:ci lighthouse && audit-ci --low --whitelist acorn minimist",
```

Comment:
Are we sure we want this to be running on ci immediately? Can we perhaps remove this for now and if everything goes fine with using and running `npm run test` locally for the next day or so we can add it onto `test:ci` in a separate PR?

Reply:
I would say that Jonathan and I have been running `npm run test` locally extensively for the past few days, and this PR has also had numerous runs through the Jenkins & Travis pipelines already  - e.g. https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/simorgh/activity?branch=react-testing-library-integration-tests and https://travis-ci.org/github/bbc/simorgh/branches?utm_medium=notification&utm_source=github_status (scroll down to look for the  react-testing-library-integration-tests).  

One of the things we discussed on the last integration tests call was that we should get it running in CI, but if others agree, then we can move it to another PR.

Reply:
I'd vote to add them to CI if you are confident they are reliable, if they turn out not to be they can be backed out with a fairly easy PR. Including them in checks allows us to gain immediate value from this work 😄

Reply:
I think running on CI now is fine. As others have said if we do see any problems it can be fixed very quickly in another PR.

Reply:
Sure, happy to go ahead. Thanks for everyone's replies.